scapy/contrib/http2.py

```diff
@@ -44,6 +44,7 @@
     class Sized(object):
         pass
 
+import scapy.base_classes as base_classes
```

Comment:
Why do you need that?

Reply:
Since the latest version, for some reason, they require docstring types to be imported (see the outdated comment above). There is a conflict between pycodestyle (flake8) that requires it, and pylint that refuses it (codacy)

Reply:
OK. From the Python interpreter point of view, this is an unused import, so I tend to think that flake8 is right here... What do you think?

Reply:
Fair enough. I've added a noqa for now, so that flake8 leaves us alone